.github/workflows/ci.yml

```diff
@@ -276,7 +276,9 @@ jobs:
         with:
           filters: |
             src:
-              - 'crates/oxc_ast/src/ast/**'
+              - 'crates/oxc_ast/src/ast/*' # single star is intentional
+              - 'crates/oxc_syntax/src/number.rs'
+              - 'crates/oxc_syntax/src/operator.rs'
```

Comment:
Should we also add `crates/oxc_ast/src/generated/*`? That'd catch if someone edits the generated files directly.

Reply:
Seems like a good idea, I'll add it in.